### PR TITLE
Refactor waitHelpers.js

### PR DESCRIPTION
**Pull request by Uzaeir**
# Description

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: step_definitions/tasks/createTask_steps.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ When(/^.* clicks on Student-submissions-visible-to-each-other checkbox$/, async
 });
 
 When(/^.* clicks on task with name '(.*)'$/, async function (taskName) {
-    await TASKListPage.sortTasksLastEdited();
+    //await TASKListPage.sortTasksLastEdited(); // Don't know what's the function of this line here? @Denis && @Yulia
```
[the remark this answers is not in the document]
Comment (yuliashikhareva) [reply step_definitions/tasks/createTask_steps.js:47]: I would say it is needed.

**Comment by yuliashikhareva** [comment]
there were some changes in Nuxt client (e.g Feature/bc 5 completed task) so that we have to adjust the tests in one extra PR, let it successfully run, merge into main and then merge changes into this branch. It should work.

File: runtime/helpers/waitHelpers.js [hunk 2]
```diff
@@ -201,9 +201,10 @@ async function waitUntilScriptResultIsTrue(script, timeoutMsg, timeout = pageLoa
 async function waitAndSetValue(selectorOrElement, value, timeout = setValueTimeout) {
 	await waitUntilElementIsVisible(selectorOrElement);
 	const element = await waitUntilElementIsEnabled(selectorOrElement);
-	await element.setValue(value);
+	await element.setValue(' ' + value);
```
[the remark this answers is not in the document]
Comment (yuliashikhareva) [reply runtime/helpers/waitHelpers.js:204]: The tests do not run if we add space here. 
Bildschirmfoto 2021-08-30 um 10 47 45

**Review by yuliashikhareva — CHANGES_REQUESTED** [review]

**Comment by yuliashikhareva** [comment]
The last test runs were fine, so I think there is no need to adjust anything in waitHelper methods. This change breaks the tests.

File: runtime/helpers/waitHelpers.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Uzaeir) [reply runtime/helpers/waitHelpers.js:204]: > why is the space required here? for me it not seems that the error is based here when a value is truncated...

Well I saw when I run the test in my local. It fails. Here you can see the screenshot from the failed test that text is just truncated the initials. Also happened at comments and other tabs. 
Download
Screenshot (14)
Screenshot (15)
Screenshot (13)

**Pull request closed without merging**